auth-web/src/resources/BreadcrumbResources.ts

```diff
@@ -1,19 +1,23 @@
 import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
+import ConfigHelper from '@/util/config-helper'
 
-export const HomeBreadCrumb: BreadcrumbIF = {
-  text: 'Business Registry Home',
-  to: { name: 'home' },
-  href: `${sessionStorage.getItem('AUTH_WEB_URL')}`
+export const MarketingHomeBreadcrumb: BreadcrumbIF = {
+  text: 'BC Registries and Online Services',
+  href: `${ConfigHelper.getValue('BCROS_HOME_URL')}`
 }
 
-export const DashboardBreadcrumb: BreadcrumbIF = {
+export const DashboardHomeBreadcrumb: BreadcrumbIF = {
+  text: 'BC Registries Dashboard',
+  href: `${ConfigHelper.getValue('BCROS_HOME_URL')}dashboard`
+}
```

Comment:
The code is fine but I want the breadcrumb names to be really clear: there are multiple "marketing" pages" and "dashboard" pages, etc.

I suggest:
"BcrosHomeBreadcrumb" (or BcRegistryDashboard ?)
"BcRegistriesDashboardBreadcrumb"
"MyBusinessRegistryBreadcrumb"

Reply:
See comment on original naming convention comment. Are those names ok?

Reply:
RegistryHomeBreadcrumb? I think you mentioned this one already 😁

Reply:
ok

Reply:
Updated!